recording_server/src/meeting/meet.ts

```
@@ -519,216 +477,159 @@ async function clickWithInnerText(
     for (let i = 0; i < maxAttempts; i++) {
         try {
             // Dump the page content to log for analysis
             if (i === 0) {
-                console.log(
-                    'Page content preview:',
-                    await page.content().then((c) => c.slice(0, 500) + '...'),
-                )
-
+                console.log('Page content preview:', await page.content().then(c => c.slice(0, 500) + '...'))
+                
                 // Log visible buttons for debugging
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)